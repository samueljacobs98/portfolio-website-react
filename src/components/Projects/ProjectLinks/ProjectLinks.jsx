import github from "../../../assets/svgs/github.svg";
import preview from "../../../assets/svgs/preview.svg";
import "./ProjectLinks.scss";

const ProjectLinks = () => {
  return (
    <div className="project-links">
      <div className="project-links__container">
        <p className="project-links__text">Code:</p>
        <img className="project-links__link" src={github} alt="" />
      </div>
      <div className="project-links__container">
        <p className="project-links__text">Code:</p>
        <img className="project-links__link" src={preview} alt="" />
      </div>
    </div>
  );
};

export default ProjectLinks;
