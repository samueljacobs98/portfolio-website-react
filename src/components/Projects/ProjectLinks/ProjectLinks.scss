@use "../../../assets/scss/variables" as *;

.project-links {
  background-color: $colour-background;
  opacity: 0.75;
  display: flex;
  justify-content: space-between;
  gap: 30px;
  border: 2px solid $color-white;
  padding: 10px 15px;
  &__container {
    display: flex;
    align-items: center;
    justify-content: center;
    .project-links__link:hover {
      cursor: pointer;
    }
  }
}
