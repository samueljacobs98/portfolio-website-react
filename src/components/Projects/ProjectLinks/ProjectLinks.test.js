import { render } from "@testing-library/react";
import ProjectLinks from "./ProjectLinks";

test("should match snapshot", () => {
  const links = { github: "", deployed: "" };
  const { container } = render(<ProjectLinks links={links} />);
  expect(container).toMatchSnapshot();
});
