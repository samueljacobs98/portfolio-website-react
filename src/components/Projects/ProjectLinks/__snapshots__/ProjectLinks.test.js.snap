// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <div
    class="project-links"
  >
    <a
      class="project-links__container"
      href=""
      rel="noreferrer"
      target="_blank"
    >
      <p
        class="project-links__text"
      >
        Code:
      </p>
      <img
        alt=""
        class="project-links__link"
        src="github.svg"
      />
    </a>
  </div>
</div>
`;
